hoprd/db/entity/src/types.rs

```diff
@@ -1,8 +1,9 @@
-use serde::{Deserialize, Serialize};
+use serde::Serialize;
 use std::fmt::{self, Display, Formatter};
 
 /// Represents an alias
-#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[cfg_attr(feature = "serde", derive(Serialize))]
```

Comment:
You possibly want to conditionally derive `Deserialize` here as well, as it used to be.